tests/acceptance/bootstrap/WebDav.php

```diff
@@ -1755,22 +1753,9 @@ public function userUploadsAFileInChunk(
 			$noOfChunks,
 			"What does it mean to have $noOfChunks chunks?"
 		);
-		// use the chunking version that works with the set DAV version
-		if ($chunkingVersion === null) {
-			if (\in_array($this->currentDAVPath, [WebDavHelper::DAV_VERSION_OLD, WebDavHelper::DAV_VERSION_SPACES])) {
-				$chunkingVersion = "v1";
-			} else {
-				$chunkingVersion = "v2";
-			}
-		}
-		$this->useSpecificChunking($chunkingVersion);
-		Assert::assertTrue(
-			WebDavHelper::isValidDavChunkingCombination(
-				$this->getDavPathVersion(),
-				$this->chunkingToUse
-			),
-			"invalid chunking/webdav version combination"
-		);
+
+		// use chunking version 1 as default, since version 2 uses "remote.php/dav/uploads" endpoint and it doesn't exist in oCIS
+		$this->chunkingToUse = 1;
```

Comment:
we need to do the proper cleanup. Get rid of `chunkingToUse` but that would be in coming PRs

Reply:
Issue for cleanup: https://github.com/owncloud/ocis/issues/10596